epub33/rs/index.html

```diff
@@ -2017,10 +2017,13 @@ <h5>Skippability</h5>
 					<h5>Escapability</h5>
 
 					<p>
-						<span id="mol-escaping-support">Reading systems SHOULD allow escaping of nested structures. </span>
-						<span id="mol-escaping-structure">Reading systems MUST determine the start of nested structures
-							by the value of the [^/epub:type^] attribute and SHOULD offer users the option to skip
-							playback of that structure and resume with whatever content comes after it.</span>
+						<span id="mol-escaping-support">While playing media overlays, reading systems SHOULD offer users
+							the option to leave ("escape") <a data-cite="epub-33#sec-escapability">escapable
+								structures</a> [[epub-33]], which are determined by the presence of an [^/epub:type^]
+							attribute [[epub-33]] with a value from the escapable structures list.</span>
+						<span id="mol-escaping-structure">If the currently-playing element or an ancestor is an
+							escapable structure, and the user opts to escape out of it, then the reading system MUST
```

Comment:
Nit: you can lose some of the words in "escape out of it." Either just "escape" or "escape it" or "escape out"?

Reply:
Maybe we can do without mention of the nesting, too. Where you are inside the escapable structure doesn't matter.

> If a user opts to escape from an escapable structure, then the reading system MUST continue playback with the next sequential element after the structure.